src/dynamics/video_player/player/player.js

```diff
@@ -681,6 +684,23 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                                 console.warn(`Please set correct companion ad attribute. It can be object with locations, string with "|" character seperated or boolean`);
                             }
                         }
+                    },
+                    "change:tracktags": function() {
+                        if (this.__trackTags) {
+                            this.__trackTags.reload();
+                        } else {
+                            if (this.__video) {
+                                this.__trackTags = new TrackTags({}, this);
+                            } else {
+                                if (this.player) {
+                                    this.player.once("loaded", function() {
+                                        if (this.__video && !this.__trackTags) {
+                                            this.__trackTags = new TrackTags({}, this);
+                                        }
+                                    }, this);
```

Comment:
This part as well is old, implementation. Probably before there was few requirements and was no need for View itself. single and double `_` here was not implemented in correct way. But there could be some cases where private method has splited into small methods, and when private calling another private, it can be with `__`